src/wasm.cc

```diff
@@ -90,91 +90,31 @@ RegisterForeignFunction::RegisterForeignFunction(std::string name, WasmForeignFu
 }
 
 void WasmBase::registerCallbacks() {
-#define _REGISTER(_fn)                                                                             \
-  wasm_vm_->registerCallback(                                                                      \
-      "env", #_fn, &exports::_fn,                                                                  \
-      &ConvertFunctionWordToUint32<decltype(exports::_fn),                                         \
-                                   exports::_fn>::convertFunctionWordToUint32)
-  _REGISTER(pthread_equal);
-#undef _REGISTER
```

Comment:
On the second though, I think we should probably retain this to avoid breaking existing extensions. But ignore it for the sake of capabilities, i.e. it should be always allowed, so no stub necessary here.

Reply:
Ok, change reverted.